Lib/glyphsLib/builder/features.py

```diff
@@ -89,6 +89,22 @@ def _to_ufo_features(font, ufo=None, generate_GDEF=False, skip_export_glyphs=Non
         if feature.notes:
             lines.append("# notes:")
             lines.extend("# " + line for line in feature.notes.splitlines())
+            feature_name = re.search(
+                "(featureNames {.+};)", feature.notes, flags=re.DOTALL
+            )
+            if feature_name:
+                name = feature_name.groups()[-1]
+                lines.extend(name.splitlines())
+            else:
+                feature_name = re.search(
+                    "^Name: (.+)", feature.notes, flags=re.MULTILINE
```

Comment:
according to a comment from @schriftgestalt:

> only if the first line starts with "Name:", the rest of the line will be used

here you're matching any line in features.notes that starts with `Name:`

Reply:
Yep, that was before Georg clarified the behaviour :)